README.md

```diff
@@ -1,32 +1,34 @@
 # GitHub-cleaner
 
-+ Note: this was originally supposed to make the labs private, until I realized you can't mark forked repos as private. Oh well. Was fun writing it.
-+ Thankfully, GitHub came out with the ability to archive repos, which is not as good, but better than nothing I guess...
-+ **[Update Jan 2019]**: GitHub has now added private repos to free accounts as well! The script has been updated accordingly. If you used the previous version of the script to archive your repos see the note below.
+- Note: this was originally supposed to make the labs private, until I realized you can't mark forked repos as private. Oh well. Was fun writing it.
+- Thankfully, GitHub came out with the ability to archive repos, which is not as good, but better than nothing I guess...
+- **[Update Jan 2019]**: GitHub has now added private repos to free accounts as well! The script has been updated accordingly. If you used the previous version of the script to archive your repos see the note below.
+- **[Preview Transfer Repos API]**: GitHub add preview API - transfer repos to new owner/organization [here](https://developer.github.com/v3/repos/#transfer-a-repository). The script has been added with this new feature.
 
 ## Overview
 
-This is a small script for Flatiron students who want to clean up their GitHub accounts by marking the Flatiron labs as private.
+This is a small script for Flatiron students who want to clean up their GitHub accounts by marking the Flatiron labs as private (fork repos have to dublicate) or transfer repos to other oganization.
 
 ### Before you start
 
-This script works by filtering your repos and looking for the `-v-000` flag, it will not archive repos not containing the flag. If there are any labs you want to keep unarchived (for example, I know I wanted to keep my TTT with AI in my GitHub), you will have to manually go in to your GitHub account and unarchive it.
+This script works by filtering your repos and looking for the `SEARCH_TERM` flag, it will not archive repos not containing the flag. If there are any labs you want to keep unarchived (for example, I know I wanted to keep my TTT with AI in my GitHub), you will have to manually go in to your GitHub account and unarchive it.
 
 Before you start you will have to obtain a personal token from GitHub, you can get it by going [here](https://github.com/settings/tokens) and clicking on the "Generate New Token" button, make sure to enable the "Repo" scope.
 
 Then go to `index.js` and add your GitHub username to the `getUsername()` function and your token to the `getToken()` function (both are near the bottom of the file).
 **Rememebr to delete your token before making any commits to the repo**
+**OR add `const TOKEN_ID USER_ID ORG_ID` in a separate `config.js` non-tracking file**
```

Comment:
Maybe add a sample config.js file?

A pattern I've seen in other repos is to create a file called something like `sample-config.js` which does get checked in to version control, and contains placeholders for all of the consts. This file serves as a sort-of template (so users can know which consts we are expecting to find). You can then update the instructions to tell users to copy the template (or just to rename it to `config.js` and add their credentials and search terms.